contracts/modules/credit/SpigotedLine.sol

```diff
@@ -17,24 +17,25 @@ contract SpigotedLine is ISpigotedLine, LineOfCredit, ReentrancyGuard {
 
     ISpigot public immutable spigot;
 
-    // 0x exchange to trade spigot revenue for credit tokens for
+    // 0x exchange venue to trade revenue tokens from a Spigot for credit tokens
     address payable public immutable swapTarget;
 
-    // amount of revenue to take from spigot if line is healthy
+    // % of revenue tokens to take from Spigot if the Line of Credit  is healthy
     uint8 public immutable defaultRevenueSplit;
 
-    // credit tokens we bought from revenue but didn't use to repay line
+    // credit tokens bought from revenue tokens but not yet withdrawn by Lenders /used to repay debt
     // needed because Revolver might have same token held in contract as being bought/sold
```

Comment:
we should remove "Revolver" here and replace with "Line of Credit" since we changed name of contracts

Reply:
done in new pr